From 38310827667f5455a5a961b8076853b6d15f4e00 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:59:11 -0400
Subject: [fs] btrfs: remove unused mnt from send_ctx

Message-id: <1407179109-13714-65-git-send-email-sandeen@redhat.com>
Patchwork-id: 86999
O-Subject: [RHEL7.1 PATCH 064/422] btrfs: remove unused mnt from send_ctx
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: David Sterba <dsterba@suse.cz>

commit a8d89f5ba0e17622cde8f5ac48ef745a9fb1e13b
Author: David Sterba <dsterba@suse.cz>
Date:   Mon Dec 16 17:34:14 2013 +0100

    btrfs: remove unused mnt from send_ctx

    Unused since ed2590953bd06b892f0411fc94e19175d32f197a
    "Btrfs: stop using vfs_read in send".

    Signed-off-by: David Sterba <dsterba@suse.cz>
    Signed-off-by: Josef Bacik <jbacik@fb.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/send.c b/fs/btrfs/send.c
index 8230d11..e98c9bc 100644
--- a/fs/btrfs/send.c
+++ b/fs/btrfs/send.c
@@ -88,8 +88,6 @@ struct send_ctx {
  u64 cmd_send_size[BTRFS_SEND_C_MAX + 1];
  u64 flags; /* 'flags' member of btrfs_ioctl_send_args is u64 */
 
- struct vfsmount *mnt;
-
  struct btrfs_root *send_root;
  struct btrfs_root *parent_root;
  struct clone_root *clone_roots;
@@ -4851,8 +4849,6 @@ long btrfs_ioctl_send(struct file *mnt_file, void __user *arg_)
   goto out;
  }
 
- sctx->mnt = mnt_file->f_path.mnt;
-
  sctx->send_root = send_root;
  sctx->clone_roots_cnt = arg->clone_sources_count;
 
-- 
1.7.1